app/models/micropost.rb

```diff
@@ -6,6 +6,7 @@ class Micropost < ApplicationRecord
   has_one_attached :image
 
   scope :order_post, ->{order created_at: :desc}
+  scope :user_post, ->(id){where user_id: id}
```

Comment:
```suggestion
  scope :by_user_id, ->user_id{where user_id: user_id}
```

cho scope xuống dưới block validate nhé